Vagrantfile

```diff
@@ -0,0 +1,36 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/xenial64"
```

Comment:
Heroku Cedar-14 is essentially Ubuntu 14.04, so that might be a more optimal target for testing. The in-beta Heroku-16 will be Xenial though.